pennylane_qiskit/runtime_devices.py

```diff
@@ -162,11 +165,9 @@ def batch_execute(self, circuits):
             program_inputs[kwarg] = self.kwargs.get(kwarg)
 
         # Specify the backend.
-        options = {"backend_name": self.backend.name()}
+        options = {"backend": self.backend.name}
         # Send circuits to the cloud for execution by the sampler program.
-        job = self.provider.runtime.run(
-            program_id="sampler", options=options, inputs=program_inputs
-        )
+        job = self.runtime_service.run(program_id="sampler", options=options, inputs=program_inputs)
```

Comment:
Why is the sampler still available as a runtime program? I think we have to make it a primitive (in the future)

Reply:
my guess is that "primitives" are still just runtime programs and they won't explicitly remove support for this. I did it to reduce the amount of change in this PR, because a primitive does not map 1:1 to a device, but here we call the sampler a "device". I'll make a story for this change, but I'm not sure how urgent it is.